analyses/cellranger-analysis/util/run_cellranger.py

```diff
@@ -0,0 +1,155 @@
+parser.add_argument('--r2_length', help='Optional. Limit the length of the input R2 sequence to the first N bases, where N is a user-supplied value. Trimming occurs before sequencing metrics are computed and therefore, limiting R2 read length may affect Q30 scores (true, false).', type=int)
+parser.add_argument('--include_introns', help='Optional. Set to false to exclude intronic reads in count. Including introns in analysis is recommended to maximize sensitivity. Refer to the guide on including introns for gene expression analysis for comprehensive recommendations. Default: true (in Cell Ranger v7.0 and later, intronic reads are counted by default)', action="store_true")
+parser.add_argument('--chemistry', help='Optional. Assay configuration. By default the assay configuration is detected automatically, which is the recommended mode. You should only specify chemistry if there is an error in automatic detection or for some special cases.', type=str)
+parser.add_argument('--check_library_compatibility', help='Optional. This option allows users to disable the check that evaluates 10x Barcode overlap between libraries when multiple libraries are specified (e.g., Gene Expression + Antibody Capture). Setting this option to false will disable the check across all library combinations. We recommend running this check (default), however if the pipeline errors out, users can bypass the check to generate outputs for troubleshooting (true, false; default: true).', type=str)
+parser.add_argument('--nosecondary', help='Optional. Adding --nosecondary to the command disables the generation of secondary analysis results, such as clustering and dimensionality reduction (tSNE/UMAP) and the .cloupe output file. This flag does not require a true or false value to be specified.', action="store_true")
+parser.add_argument('--min_crispr_umi', help='Optional. Set the minimum number of CRISPR guide RNA UMIs required for protospacer detection. Default: 3 UMIs (in Cell Ranger v8.0 and later). If a lower or higher sensitivity is desired for detection, this value can be customized according to specific experimental needs. Applicable only to datasets that include a CRISPR Guide Capture library.', type=int)
+parser.add_argument('--file', help='Optional. Tsv file containing the following	header and corresponding info (#ID SAMPLE FASTQ) for batch submission.', type=argparse.FileType('r'))
+
+
+#Converts argument strings to objects and assigns them as attributes of the namespace; e.g. --id -> args.id
+args = parser.parse_args()
+
+#Accessing reference genome based on requested short name of reference genome
+#Accessing reference genome based on requested short name of reference genome
+if args.transcriptome == "GRCh38":
+	args.transcriptome = "/research/dept/dnb/core_operations/Bioinformatics/common/ReferenceGenomes/homo_sapiens/GRCh38/downloads/refdata-gex-GRCh38-2024-A/"
+elif args.transcriptome == "mm10":
```

Comment:
A slight technicality here, GRCm39 and mm10 are not actually the same reference (GRCm39 is the most recent mouse reference, mm10 is the previous one). So really the reference you are pointing to is GRCm39 (aka mm39), not mm10

Reply:
```suggestion
elif args.transcriptome == "GRCm39":
```